x/clp/test/test_common.go

```diff
@@ -50,9 +50,14 @@ func CreateTestApp(isCheckTx bool) (*simapp.SimApp, sdk.Context) {
 }
 
 func CreateTestAppClp(isCheckTx bool) (sdk.Context, keeper.Keeper) {
+	ctx, app := GetSimApp(isCheckTx)
+	return ctx, app.ClpKeeper
+}
+
+func GetSimApp(isCheckTx bool) (sdk.Context, *simapp.SimApp) {
```

Comment:
This function seems redundant, and no one else is calling it.  What is its purpose?

Reply:
I just broke down the CreateTestAppClp , into two functions ,for code clarity . Might use just the simapp for some future tests